modmail/extensions/utils/error_handler.py

```diff
@@ -0,0 +1,195 @@
+import logging
+import re
+import typing
+
+import discord
+import discord.errors
+from discord.ext import commands
+
+from modmail.bot import ModmailBot
+from modmail.log import ModmailLogger
+from modmail.utils.cogs import BotModes, ExtMetadata, ModmailCog
+from modmail.utils.extensions import BOT_MODE
+
+
+logger: ModmailLogger = logging.getLogger(__name__)
+
+EXT_METADATA = ExtMetadata()
+
+ERROR_COLOUR = discord.Colour.red()
+
+ERROR_TITLE_REGEX = re.compile(r"(?<=[a-zA-Z])([A-Z])(?=[a-z])")
+
+ANY_DEV_MODE = BOT_MODE & (BotModes.DEVELOP.value + BotModes.PLUGIN_DEV.value)
```

Comment:
Is there not a better way to do this from ExtMetadata?

Reply:
Perhaps ExtMetadata or BotModes should have an `any_dev_mode` property?

Reply:
I don't want to add that to the metadata system, also metadata got a bit of a refactor in GH-69.

What I could add to the cog system is this constant, and just compute it there instead.